test/test_itensorfunction.jl

```diff
@@ -42,12 +42,22 @@ end
     ψ_fx = const_itn(s; c)
 
     x = 0.5
-    ind_to_ind_value_map = calculate_ind_values(ψ_fx, x)
-
-    fx_x = calculate_fx(ψ_fx, x)
+    fx_x = calculate_fx(ψ_fx, x; alg="exact")
     @test fx_x ≈ c
   end
 
+  @testset "test const w/ linkdims" begin
```

Comment:
Lets just make this the `test const` test set and remove the above. We can test both the `linkdim = 1` and `linkdim = 4` in the same test set.